### PR TITLE
improve postgres install instructions

### PR DESCRIPTION
**Pull request by loisgh**
- No username or password was included in the database.yml.  There was nothing to let someone know
that these parameters might be needed on their system.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ Prior to running 'bin/setup', it may be necessary to launch the postgres server
 
 To get started, run `bin/setup`
 
+if bin/setup fails check your database.yml file and make sure that you have postgres credentials (i.e.,username:<uname>, password:<pw>)
```
Comment (afeld) [inline CONTRIBUTING.md]: Maybe we could provide more details about what sort of error the person would see? There are a lot of things that happen in `bin/setup`, so would want to narrow it down a bit. Minor: Also, mind capitalizing the first letter?

**Comment by afeld** [comment]
Looks good to me! Will defer to the team for merging.

**Comment by jessieay** [comment]
This is great - thanks! Mind updating to use the markdown style we are using for blocks of markdown in the rest of this file? (prepend and append with three backticks so you don't need to indent)

**Comment by loisgh** [comment]
Sure I'll take care of that.

Lois

**Comment by loisgh** [comment]
This is done.

Thanks

Lois

**Comment by jessieay** [comment]
hi @loisgh sorry for the back and forth but the formatting is kind of weird now. the raw code should look like this:
screen shot 2016-07-26 at 1 55 50 pm

the way you have it now renders kind of weirdly if you look at it rendered at https://github.com/loisgh/dolores-landingham-bot/blob/14baa88ff6bfde2f133eef1da22a4c6cb333a13f/CONTRIBUTING.md

**Comment by loisgh** [comment]
Hi Jessie,

No worries.  I'm happy to fix it.

Lois

**Comment by jessieay** [comment]
Merged to master at 23c1f52ca7d6659e7bdb3ffd094d25891cf8d578 -- thank you!! 🌈

**Pull request closed without merging**